zqd/ingest/log.go

```diff
@@ -79,6 +79,7 @@ func ingestLogs(ctx context.Context, pipe *api.JSONPipe, s *space.Space, req api
 		cfg.JSONTypeConfig = req.JSONTypeConfig
 		cfg.JSONPathRegex = DefaultJSONPathRegexp
 	}
+	var totalSize int64
 	for _, path := range req.Paths {
 		sf, err := detector.OpenFile(zctx, path, cfg)
```

Comment:
@alfred-landrum regarding your question about getting bytes read from the reader. I would suggest opening the file manually here and wrapping the file in some `io.Reader` struct that increments some counter based off the bytes pulled off from `Read([]byte)`.

Reply:
Then just doing `detector.NewReader` instead of `detector.OpenFile`.

Reply:
I've adjusted this to get the actual bytes read; it was more complex than calling NewReader, as the I wanted to leave the json configuration in the detect package. PTAL.